app/src/main/java/com/ataulm/whatsnext/search/SearchActivity.kt

```diff
@@ -79,3 +92,27 @@ private fun SearchActivity.injectDependencies() {
             .build()
             .inject(this)
 }
+
+private class PopularFilmsThisWeekAdapter(
```

Comment:
Search should also be paged probably 🤔 I think we assume that the film we're looking for will be in the first page anyway